### PR TITLE
Use a task to set up an additional NFS repository

### PR DESCRIPTION
**Pull request by poncovka**
Set up an additional NFS repository with a task. Collect generated mount points,
so we can unmount them when needed. This is just a temporary solution, because
the source management should be eventually handled by DBus source objects.

### CODE REVIEW

File: pyanaconda/payload/dnf/payload.py [hunk 1]
```diff
@@ -433,6 +434,39 @@ def _is_existing_repo_configuration(self, data):
...
+                url=self._dnf_manager.substitute(data.url)
+            )
+            mount_point = task.run()
+            data.url = "file://" + mount_point
+
+    def _get_mount_point(self, *paths):
+        """Generate a mount point from specified paths."""
+        mount_point = join_paths(*paths)
+        self._mount_points.append(mount_point)
+        return mount_point
```
Comment (VladimirSlavik) [inline pyanaconda/payload/dnf/payload.py:461]: That doesn't really generate the mount point, only its path, I think. So maybe call it `_get_mount_point_path`, and change the docstring for that?

**Review by VladimirSlavik — APPROVED** [review]
Thank you, looks good to me.

File: pyanaconda/payload/dnf/payload.py (hunk 1 above)
Comment (poncovka) [reply pyanaconda/payload/dnf/payload.py]: I have changed it, however, these methods are temporary.

**Pull request merged**